federation/client/clientset_generated/federation_release_1_5/typed/extensions/v1beta1/generated_expansion.go

```diff
@@ -16,6 +16,8 @@ limitations under the License.
 
 package v1beta1
 
+type DeploymentExpansion interface{}
```

Comment:
This should have a Rollback method: https://github.com/kubernetes/kubernetes/blob/a44d2f8009ab76d190ad9572e4aca661d5b6c8a3/pkg/client/clientset_generated/release_1_5/typed/extensions/v1beta1/deployment_expansion.go#L22.

Add a deployment_expansion.go file (same as the one mentioned above)